localization/english/culture/wc_cultural_innovations_l_english.yml

```diff
@@ -0,0 +1,44 @@
+﻿l_english:
+ epidemic_conversion_bonus:0 ""Promote Culture" [councillor_task|E] Speed: #P +50%#!"
+ 
+ ##############
+ # Tribal Era #
+ ##############
+
+ innovation_amphibious:0 "Amphibious"
+ innovation_amphibious_desc:0 "We surface on land only when we please, but being natives to the water we can travel the seas at ease."
+
+ innovation_sea_faring:0 "Sea Faring"
+ innovation_sea_faring_desc:0 "Living on the coasts gave us a natural advantage in not only taming, but conquering the seas and those who dare to challenge us."
+
+ innovation_lorewalking:0 "Lorewalking"
```

Comment:
It looks like you're joking or something...
What is this file doing on this branch?